dwave/optimization/mathematical.py

```diff
@@ -583,6 +585,27 @@ def put(array, indices, values):
     """
     return Put(array, indices, values)
 
+def rint(x: ArraySymbol) -> RoundInt:
+    r"""Return an element-wise rint on the given symbol.
```

Comment:
Don't need the leading `r` in this case. Also, could use a bit more spacing. `sphinx` sometimes gets confused without it.